packages/core/src/components/table/table-component-expandable-rows.stories.tsx

```diff
@@ -124,6 +124,36 @@ export default {
       },
       if: { arg: 'noMinWidth', eq: true },
     },
+    flexDisplay: {
```

Comment:
Instead of adding these props on the row, can't we have the users slot in an element which they control the layout of. All our expandable-row should do is to overflow correctly? So we could add a simple `overflow` prop on the row.

Reply:
I'm absolutely open to this approach. What do you think @timrombergjakobsson  @nathalielindqvist  @Lunkan89  @mistermalm  ?

Reply:
Good suggestion!

Reply:
I might be wrong, but is this issue fixed by @mistermalm under https://github.com/scania-digital-design-system/tegel/pull/721 ? 

Please, double check, I am not 100% sure. Maybe we just missed to close GitHub issue?

Reply:
Hm yes, it seems so, good eyes @theJohnnyMe.